### PR TITLE
Throw if product info is null

### PR DESCRIPTION
**Pull request by Y-Sindo**

### CODE REVIEW

File: src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs [hunk 1]
```diff
@@ -124,7 +124,14 @@ public static IServiceCollection AddUserAgent(this IServiceCollection services,
...
             }
 
-            return services.PostConfigure<ServiceManagerOptions>(o => o.ProductInfo += userAgent);
+            return services.PostConfigure<ServiceManagerOptions>(o =>
+            {
+                if (o.ProductInfo == null)
+                {
+                    throw new InvalidOperationException("Product info is null");
+                }
+                o.ProductInfo += o.ProductInfo + userAgent;
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs:133]: typo?

Comment (Y-Sindo) [reply src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs:133]: Which word?

Comment (vicancy) [reply src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs:133]: o.ProductInfo += o.ProductInfo

Comment (Y-Sindo) [reply src/Microsoft.Azure.SignalR.Management/DependencyInjectionExtensions.cs]: Got it, I was checking spelling error...

**Review by vicancy — APPROVED** [review]

**Pull request merged**